plenoptic/tools/conv.py

```diff
@@ -92,8 +92,8 @@ def blur_downsample(x,filtname = 'binom5', step=(2, 2)):
     return correlate_downsample(x, filt=np.outer(f, f), step=step)
 
 
-def upsample_blur(x, step=(2, 2)):
-    f = pt.named_filter('binom5')
+def upsample_blur(x, filtname ='binom5', step=(2, 2)):
+    f = pt.named_filter(filtname)
```

Comment:
We should add a docstring here and include the possible filters (just copying over the relevant part from `pt.named_filter` docstring

Reply:
I still think this is true, but all the functions in `conv.py` need their docstrings revisited, so I think this can wait.